Library/Homebrew/rubocops/cask/no_overrides.rb

```diff
@@ -9,7 +9,7 @@ class NoOverrides < Base
 
         # These stanzas can be overridden by `on_*` blocks, so take them into account.
         # TODO: Update this list if new stanzas are added to `Cask::DSL` that call `set_unique_stanza`.
-        OVERRIDEABLE_METHODS = [
+        OVERRIDABLE_METHODS = [
```

Comment:
This I'm not so on board with. After a semi-brief trip down the [-able suffix rabbit hole](https://english.stackexchange.com/q/55193), I believe the -e should be preserved here, since it serves to make the previous vowel long. Without it, the word looks like "over-rid-able" instead of "over-ride-able".

Reply:
I don't feel super strongly about this one. This is what `typos-cli` suggested. One thing I did notice is that "overridable" has ~869,000 results on Google while "overrideable" has ~57,300 results and it offers "Did you mean: overridable" when searching it.

Reply:
Agreed with @EricFromCanada too.